packages/kernels/linux-rpi/collection.yaml

```diff
@@ -0,0 +1,9 @@
+packages:
+  - name: "linux-rpi"
+    category: "kernels"
+    version: "rpi-6.6.y"
```

Comment:
what's the y for?

Reply:
thats what they call their branch names lol . Instad of 6.6.x they use the y for some reason :D

Maybe they did rpi-6.x.y before?